.github/workflows/ci.yml

```diff
@@ -557,6 +570,7 @@ jobs:
     outputs:
       have_azure_secrets: ${{ steps.check-secrets.outputs.have_azure_secrets }}
       have_databricks_secrets: ${{ steps.check-databricks-secrets.outputs.have_databricks_secrets }}
+      have_gcp_secrets: ${{ steps.check-secrets.outputs.have_gcp_secrets }}
```

Comment:
GCS suite looks not running. Probably, this should be `steps.check-gcp-secrets.outputs.have_gcp_secrets`.

Reply:
It's fixed now.
<img width="1007" alt="Screen Shot 2022-08-23 at 13 52 56" src="https://user-images.githubusercontent.com/6237050/186072854-954e237b-35f4-4783-bbe7-0fc1f1d01779.png">